requirements.txt

```
@@ -1,13 +1,13 @@
-colorama>=0.4.5
+colorama>=0.4.6
```

```suggestion
colorama>=0.4.5
```
possibly broken on linux?

---

Nothing to do with colorama.

`logger.info()` simply does not print to stdout for me, but `print()` and `logger.error()` does. `host.yaml` says `loglevel: "info"`, so that should be fine? Any ideas?

---

No. I guess you could breakpoint into the log function and see where it goes wrong? Again, on my system it all works fine.

---

Turns out if I add
```python
logger.setLevel(logging.INFO)
```
right after
```python
logger = logging.getLogger("Client")
```
I get messages. So it seems like `logging.getLogger()` has a separate minimum logging level?